defaultFrameworkAutomation-main/src/test/java/test/authorization/RegistrationTest.java

```diff
@@ -0,0 +1,34 @@
+package test.authorization;
+
+import org.testng.annotations.Test;
+import pages.authorization.AuthorizationPage;
+import pages.register.RegistrationPage;
+import services.authorization.AuthorizationService;
+import services.main.MainService;
+
+import static com.codeborne.selenide.Condition.visible;
+
+public class RegistrationTest {
+    private final AuthorizationService authorizationService = new AuthorizationService();
+    private final AuthorizationPage authorizationPage = new AuthorizationPage();
+    private final MainService mainService = new MainService();
+    private final RegistrationPage registrationPage = new RegistrationPage();
+
+
+    @Test
+    public void checkRegistration() {
+        String name = "bfbka";
+        String email = "kt@gmail.com";
+        String password = "seoy2199";
+
+        authorizationService.openAutorizationPage();
+        authorizationPage.getLinkRegister().shouldBe(visible).click();;
```

Comment:
я уже говорил, что конструкция shouldBe(visible) всегда должна быть с because 
![image](https://github.com/user-attachments/assets/da2dd735-6d8a-4fa7-b433-413a7ef5baba)